src/compose/volumes.coffee

```diff
@@ -0,0 +1,108 @@
+Promise = require 'bluebird'
+_ = require 'lodash'
+fs = Promise.promisifyAll(require('fs'))
+ncp = Promise.promisify(require('ncp').ncp)
+rimraf = Promise.promisify(require('rimraf'))
+exec = Promise.promisify(require('child_process').exec)
+ncp.limit = 16
+
+logTypes = require '../lib/log-types'
+migration = require '../lib/migration'
+constants = require '../lib/constants'
+
+ENOENT = (err) -> err.code is 'ENOENT'
+
+module.exports = class Volumes
+	constructor: ({ @docker, @logger }) ->
+
+	format: (volume) ->
+		appId = volume.Labels['io.resin.app_id']
```

Comment:
shouldn't this be converted to a number?

Reply:
Now that we're using numbers for ids, yes. Done.